docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -63,21 +63,37 @@ If the battery is low, the connection between your station and the M5 gateway wi
 
 ## Sensor Problems
 
-Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
-There are several reasons that you may experience this error, which includes:
+We have discovered that the weather data packets received, did not meet our quality standards. 
+To ensure the accuracy of the information sent by our weather stations, we evaluate all data packets using our Quality of Data (QoD) algorithm. For more information about this algorithm, please refer to this [QoD](/project/quality-of-data). 
```

Comment:
1. Let's make `Quality of Data (QoD) algorithm` hyperlinked pointing to the docs algorithm.
2. to this [QoD] ---> [here]